DirectEmployers/MyJobs
# [PD-3261] export current tab

## What's this PR do?
PR fixes the exporting for the current tab that the user is on inside of analytics

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of any of the analytics reports

## How should this be manually tested?
Pull down repository. Go into any one of the analytics and create more tabs by drilling down. Switch the tabs and export the data which should be the data for the current tab

## Any background context you want to provide?
